web-bundle/src/main/java/com/graphhopper/customspeeds/CustomSpeedsUtils.java

```diff
@@ -20,6 +24,7 @@ public class CustomSpeedsUtils {
     public static final String CUSTOM_SPEED_FILE_CONFIG_FIELD = "custom_speed_file";
     private static final String CUSTOM_SPEED_OSM_WAY_ID_COL_NAME = "osm_way_id";
     private static final String CUSTOM_SPEED_MAX_SPEED_KPH_COL_NAME = "max_speed_kph";
+    private static final String CUSTOM_SPEED_BWD_COL_NAME = "bwd";
```

Comment:
re: `+/-` vs. `bwd`, i like the explicit backwards column! `+/-` probably makes more sense for stable edge ids since it's part of their format, but that's not true for osm way ids so no need to bastardize them 🙂

more generally, i realized the format we use for the scenario speeds input file doesn't have to match the format we use for the profile speeds files we feed to the router. we already apply several transformations to the original inputs file to produce the profile speeds files (like partitioning the input speeds by mode). this decoupling gives us some valuable flexibility. so we could also use `bwd` for the scenario input files, but just wanted to note that we are free to use a different scheme if that turns out to be awkward for whatever reason 🙂